eclair-core/src/test/scala/fr/acinq/eclair/EclairImplSpec.scala

```diff
@@ -149,8 +149,8 @@ class EclairImplSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike with I
     assert(send3.recipientNodeId === nodePrivKey.publicKey)
     assert(send3.recipientAmount === 123.msat)
     assert(send3.paymentHash === ByteVector32.Zeroes)
-    assert(send3.routeParams.maxFeeBase === 123000.msat) // conversion sat -> msat
-    assert(send3.routeParams.maxFeePct === 4.20)
+    assert(send3.routeParams.maxFeeFlat === 123000.msat) // conversion sat -> msat
+    assert(send3.routeParams.maxFeeProportional === 4.20)
```

Comment:
This was before your PR, but I'm realizing now that the value for this test is completely absurd (420%) and users may find it very misleading...could you update that to a "normal" percentage, such as `0.15` (15%)?

Reply:
I've changed it so that `maxFeePct` is treated as a percentage.